README.rst

```diff
@@ -120,7 +120,7 @@ STIX 1.1.1 - 1.2.1 content to STIX 2.0 content::
       --validator-args VALIDATOR_ARGS
                             Arguments to pass to stix-validator. Default: --strict-
                             types Example: stix2_elevator.py <file> --validator-
-                            args "-v --strict-types -d 212"
+                            args="-v --strict-types -d 212"
```

Comment:
Is `=` actually required or is it more of a consistency thing?

Reply:
It wouldn't work for me in zsh without the `=`.

Reply:
I was able to test it using `=` without any problems on Windows command-line. The behavior is still correct.